h5py/tests/test_dataset.py

```diff
@@ -762,6 +853,9 @@ def test_create_1D(self):
         dset = self.f.create_dataset('bar', 20, maxshape=20)
         self.assertEqual(dset.maxshape, (20,))
 
+        dset = self.f.create_dataset('fun', (1,), maxshape=True)
```

Comment:
Please don't test this. It's not really intended or documented to work, it just happens because `True == 1` in Python.

Reply:
Ok, will do.